### ISSUE TITLE
[Window Previews] - A setting for be always open?

### ISSUE THREAD

**Issue by alex285** [micheleg/dash-to-dock#400]
Windows previews are awesome, but the extra click needs to open the slider makes it to slow. Can be a setting to keep it always open, perhaps changing the size etc too?

**Comment by micheleg** [comment]
Yes, considering it, or moving it to the hover menu (#273), as implemented in https://github.com/ZorinOS/zorin-taskbar (a fork of this extension by ZorinOs http://zoringroup.com/blog/2016/09/19/this-is-zorin-os-12/).

**Comment by alex285** [comment]
@micheleg I didnt try Zorin extension, I just saw the screenshots. On hover maybe be okay, but maybe gets annoying when the Dock is hidden and you press to reveal it. 

The way it works now, it has an "arrow menu" that hides/shows the thumbs. If that arrow could remember the last option and was either on or close, maybe? 

But I'm actually out of proposals! I think you should play with deep options here ..Or just start trying things and get some feedback!

**Comment by micheleg** [comment]
I made a quick "implementation" of it [77422abfd42f43926c51a6ce7bcdb7504606346b], but it was giving some glitches, therefore I haven't merged it as an option yet. 

Any opinion about if the previews should be layered out horizontally for an horizontal dock?

**Comment by alex285** [comment]
@micheleg now it is much more useful, because you have the quick access! I guess horizontal makes more sense, but vertical is nice too! 

basically is really really cool feature!

**Comment by Baltix** [comment]
@micheleg , thanks for including windows previews, but please add an option to make window thumbnails come up instead of the app title on hovering over an icon or clicking with left mouse button

micheleg commented at issue #273 :
> I've merged the branch with the feature in master [72bd2c8]. At the moment the windows previews menu is close by default.

**Comment by slacka** [comment]
The previews should be close to the dock so little mouse movement is necessary to select. Horizontal dock should have horizontal previews, while vertical dock should have vertical previews.

**Comment by marcvangend** [comment]
+1, in fact "can this always be open" was the first thought when I discovered the new feature.

> Horizontal dock should have horizontal previews, while vertical dock should have vertical previews.

I'm not sure if I agree. The horizontal dock has a vertical popup menu, so stacking the previews vertically (as it is now) is not a strange solution IMO. Perhaps we need to see and try it to figure out what works best. I guess this is especially important when you have many windows, ie. 4 or more. It might even make sense to put them in a grid (eg. 6 previews in a 2x3 grid) to make sure it doesn't fall off the screen.
